### ISSUE TITLE
Walker menu hover state on iPad <Question>

### ISSUE THREAD

**Issue by merijnorama** [hallme/scaffolding#94]
My apologies beforehand since this is not really an issue but a question, which I hope the authors might be willing to help me with:

It seems that the current (standard) implementation of the walker menu in scaffolding.io does not allow a hover state on a menu item revealing it's subitems when viewing on the iPad (it registers a click on the parent item right away and takes the user there). I was wondering what would be the best approach to solve this? I looked at numerous posts on the subject, but not sure which path to choose.

**Comment by jlavoie13** [comment]
Thanks for your question. We'll take a look and let you know what we can figure out.

**Comment by merijnorama** [comment]
Thank you for your quick reply. I was actually investigating the possibilty of adding onClick="return true" to the menu <li> items via functions.php. But so far that does not show up in the generated menu links.

I also tried adding this to scripts:

document.addEventListener("touchstart", function(){}, true);

And this to the styles:

element:hover, element:active {
-webkit-tap-highlight-color: rgba(0,0,0,0);
-webkit-user-select: none;
-webkit-touch-callout: none
}

But that did not seem to work.

**Comment by jlavoie13** [comment]
Hi, sorry for the late reply but I seem to have found a fix. It isn't perfect but maybe with some tweaking. https://snippets.webaware.com.au/snippets/make-css-drop-down-menus-work-on-touch-devices/ 

I'll add it to the theme with some more testing if it works out. Let me know if this helps you.

**Comment by merijnorama** [comment]
Great! Thank you for your reply. I will try it out as soon as you have added it to the theme.

**Comment by merijnorama** [comment]
I tried this adding it to my scripts.js, but so far I did not manage to get it working.

It probably has something to do with targeting the right CSS class, I tried `li#menu-item.menu-item-type-taxonomy.menu-item-object-category.menu-item-has-children a`

Also, the standard version of this script disables at >iOS5 while the problem seems to occur on iOS8 as well. So I think it might be better to disable `&& !iOS5` on line 8.
